aio_pika/robust_connection.py

```diff
@@ -124,7 +121,7 @@ def channel(self, channel_number: int = None,
         for number, channel in self._channels.items():
             try:
                 await channel.on_reconnect(self, number)
-            except (RuntimeError, ChannelClosed, AMQPError):
+            except (RuntimeError, AMQPError):
```

Comment:
@mosquito should we check for `CONNECTION_EXCEPTIONS` here? I mean `ConnectionError` and `PAMQP_EXCEPTIONS`.

Reply:
@mosquito ?